README.md

```diff
@@ -0,0 +1,79 @@
+gceimgutils
+===========
+
+A collection of utilities for image management in Google Compute Engine
+
+While the functionality provided by the utilities is more or less a duplication
+of functionality available with the gcloud CLI tools from the [google-cloud-sdk](https://cloud.google.com/sdk) the google-cloud-sdk does not provide consumable sources for packaging in a distribution nor is there a project for upstream contributions. This project focuses on image management only.
+
+All utilities expect the service account credentials in ~/.config/gce. The name of the credentials file is expected to match the project name followed by the .json extension.
+
+## Installation
+
+### openSUSE and SUSE Linux Enterprise
+
+```
+> zypper in python3-gceimgutils
+```
+
+### PyPI
+
+```
+> pip install gceimgutils
+```
+
+## Utilities
+
+### gcedeprecateimg
```

Comment:
Suggestion: Consider a single command or namespace. With aliyun-img-utils I chose to make a single command with subcommands. The reasoning is that it could allow for complex options. Such as a single subcommand to do the whole image publish process or a single subcommand to create an image. Rather than requiring a user to run each command one by one (upload, create, replicate etc).

Reply:
Same as above, Seems we end up one way or the other as the existing ec2imgutils has multiple commands.

Reply:
I know the main goal here based on the readme is to solve the packaging problem so in that way it makes sense to have 1-1 mapping to gcloud. But IMHO it would be helpful to have higher level functions from a user standpoint:

```
gceimgutils publish (which takes a tarball locally to a published image in GCE)
```

Which could of course be:

```
gcepublishimg
```